snapr/src/lib.rs

```diff
@@ -309,15 +290,19 @@ impl<'a> Snapr<'a> {
 
         match self.tile_fetcher {
             TileFetcher::Individual(tile_fetcher) => {
+                // Capture various fields in `self` to enable `x_y_to_tile` to automatically implement `Sync`
+                let (tile_fetcher, tile_size, height, width, zoom) =
+                    (tile_fetcher, self.tile_size, self.height, self.width, zoom);
```

Comment:
I think I am stupid because it took me way too long to figure out that `x_y_to_tile` didn't implement `Sync` because of `self`, that's a no-brainer. 🤦